tests/components/filesize/__init__.py

```diff
@@ -8,7 +8,7 @@
 TEST_FILE2 = os.path.join(TEST_DIR, TEST_FILE_NAME2)
 
 
-def create_file(path) -> None:
+async def async_create_file(path) -> None:
     """Create a test file."""
     with open(path, "w", encoding="utf-8") as test_file:
```

Comment:
We need to schedule the file call(s) to the executor. We're still in the event loop here. Probably make a function that does this block and schedule that on the executor with `hass.async_add_executor_job`.

Reply:
I put in what I think was the suggestion.